helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -489,6 +523,8 @@ service:
         {{- if eq (include "splunk-otel-collector.o11yLogsEnabled" .) "true" }}
         - splunk_hec/o11y
         {{- end }}
+        {{- end }}
+        {{- end }}
```

Comment:
One `end` is shifted a bit. And if the suggestion above is applied, it can be removed

Reply:
please remove one `{{- end }}` from here